resources/main.css

```diff
@@ -79,6 +77,15 @@ a {
 }
 
 button {
+    -webkit-appearance: none;
```

Comment:
Why -webkit- prefix?

Reply:
This was just copied over from a few lines above.
- Updated all outdate `-webkit-` prefixed CSS properties for consistency